packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -401,6 +396,10 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 						authType: nodeInfo?.authType?.name,
 					}
 				: undefined,
+			currentWorkflow: workflowDataStale.value ? workflowsStore.workflow : undefined,
+			executionData: workflowExecutionDataStale.value
```

Comment:
have we discussed this with Nik in terms of privacy?

Reply:
Not for this feature but we were discussing it when we introduced additional context in the generic error helper.

Reply:
Update: We decided to remove node data from execution results